### ISSUE TITLE
[BUG] Same multiple alert messages when switching between source chains

### ISSUE THREAD

**Issue by devpavan04** [webb-tools/webb-dapp#868]
### Overview
Same multiple alert messages are shown when switching between different source chains.

### Issue screenshot
<img width="1031" alt="Screenshot 2023-01-11 at 3 52 23 AM" src="https://user-images.githubusercontent.com/53374218/211799637-7b830eb6-e1fe-44c3-8cd3-77808002b5cd.png">

### Loom video
https://www.loom.com/share/00051b20deb049cd828e570042fd6b2d

Tested on branch https://github.com/webb-tools/webb-dapp/pull/850 merged into develop.

**Comment by devpavan04** [comment]
Closing this issue as it was not found on latest updates on `develop` branch.

Loom video:
https://www.loom.com/share/4074e9fc9a904b67879d361538cd54ba

Tested on #901 into develop.

**Comment by devpavan04** [comment]
Issue persists - https://www.loom.com/share/d727debc5fad4d348149fa03dee50400

**Comment by devpavan04** [comment]
@dutterbutter 

Issue seems to be found when something breaks (not related to notification) during development. `Change of chain` notification is triggered only once 99% of the times. Unable to reproduce this issue and find the root cause of it in the code as the notification is working correctly. Need to know if this is reproducible on your end. 

Issue not found:
https://www.loom.com/share/09b7e3c4aba74ddeaca63091fbb49d1f
